clang/bindings/python/tests/cindex/test_cursor.py

```diff
@@ -279,6 +280,90 @@ def test_is_default_method(self):
         self.assertTrue(xc.is_default_method())
         self.assertFalse(yc.is_default_method())
 
+    def test_is_deleted_method(self):
+        source = "class X { X() = delete; }; class Y { Y(); };"
+        tu = get_tu(source, lang="cpp")
+
+        xs = get_cursors(tu, "X")
+        ys = get_cursors(tu, "Y")
+
+        self.assertEqual(len(xs), 2)
+        self.assertEqual(len(ys), 2)
+
+        xc = xs[1]
+        yc = ys[1]
+
+        self.assertTrue(xc.is_deleted_method())
+        self.assertFalse(yc.is_deleted_method())
+
+    def test_is_copy_assignment_operator_method(self):
```

Comment:
You might want to add a case with inherited copy assignment operator.

Reply:
How do you mean exactly? If I have a class inherit a copy-assignment operator, there is no function declaration cursor for that class to call `is_copy_assignment_operator_method` on.

Reply:
Hm, so cursor goes over source representation of the code rather than semantics? Fine then.

Reply:
Seems like it. I just added an inheriting class to check, and the number of detected `operator=` declarations in that TU stays unchanged.